module/purpose/Soccer/data/config/webots2/Soccer.yaml

```diff
@@ -0,0 +1,9 @@
+# Controls the minimum log level that NUClear log will display
+log_level: INFO
+
+# For the robot playing without listening to GameController
+force_playing: false
+
+# The position of the robot
+# GOALIE, STRIKER, DEFENDER
+position: DEFENDER
```

Comment:
it might be easier to only overwrite position in these robot-specific config files. Then we have one spot (default config) to set force playing and log level.